deploy/compliance-operator-chart/crds/compliance.openshift.io_compliancescans_crd.yaml

```diff
@@ -74,6 +74,11 @@ spec:
                   generated from the scan, this should match the selector of the MachineConfigPool
                   you want to apply the remediations to.
                 type: object
+              priorityClass:
+                description: Defines the PriorityClass to use for launching scan related
+                  pods, if PriorityClass is invalid and not found, we will treat scansetting
+                  didn't have a PriorityClass.
```

Comment:
After looking through the other description, I think we should avoid the first person point of view (e.g. removing "we").

Perhaps:

```yaml
description: Defines the PriorityClass to use for launching scan-related pods.
  If the PriorityClass is not found, pods won't be scheduled using a PriorityClass.
```